x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -0,0 +1,329 @@
+          size: PAGE_SIZE,
+          numBuckets: 20,
+          pageIndex: tableOptions.pageIndex,
+          sortField: tableOptions.sort.field,
+          sortDirection: tableOptions.sort.direction,
+        },
+      },
+    }).then((response) => {
+      return {
+        items: response.transaction_groups,
+        totalItemCount: response.total_transaction_groups,
+        tableOptions: {
+          pageIndex: tableOptions.pageIndex,
+          sort: {
+            field: tableOptions.sort.field,
+            direction: tableOptions.sort.direction,
+          },
+        },
+      };
+    });
+  }, [
+    serviceName,
+    start,
+    end,
+    uiFilters,
+    tableOptions.pageIndex,
+    tableOptions.sort.field,
+    tableOptions.sort.direction,
+  ]);
+
+  const {
+    items,
+    totalItemCount,
+    tableOptions: { pageIndex, sort },
+  } = data;
+
+  const columns: Array<EuiBasicTableColumn<ServiceTransactionGroupItem>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnName',
+        {
+          defaultMessage: 'Name',
+        }
+      ),
+      render: (_, { name, transaction_type: transactionType }) => {
+        return (
+          <TransactionGroupLinkWrapper>
+            <EuiToolTip delay="long" content={name}>
+              <StyledTransactionDetailLink
+                serviceName={serviceName}
+                transactionName={name}
+                transactionType={transactionType}
+              >
+                {name}
+              </StyledTransactionDetailLink>
+            </EuiToolTip>
+          </TransactionGroupLinkWrapper>
+        );
+      },
+    },
+    {
+      field: 'latency',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency',
+        {
+          defaultMessage: 'Latency',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { latency }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis1"
+            compact
+            series={latency.timeseries ?? undefined}
+            valueLabel={
+              latency.value !== null
+                ? asDuration(latency.value)
+                : NOT_AVAILABLE_LABEL
+            }
+            start={parseFloat(start!)}
+            end={parseFloat(end!)}
+          />
+        );
+      },
+    },
+    {
+      field: 'traffic',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnTraffic',
+        {
+          defaultMessage: 'Traffic',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { traffic }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis0"
+            compact
+            series={traffic.timeseries ?? undefined}
+            valueLabel={
+              traffic.value !== null
+                ? asTransactionRate(traffic.value)
+                : NOT_AVAILABLE_LABEL
+            }
```

Comment:
You already handle the "not available" state in `asTransactionRate`:

```suggestion
            valueLabel={asTransactionRate(traffic.value)}
```